astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
Don't have a strong opinion, but I think simply specifying a range with upper and lower is reasonable, and where it defaults to None for either `upper_limit` or `lower_limit` indicating no limit in that direction.